### PR TITLE
Fix settings route from /personal to /user

### PR DESCRIPTION
**Pull request by jancborchardt**
Since that was changed with the move to unified settings, already in Nextcloud 12. So we should backport this too. Please review @nickvergessen @blizzz

### CODE REVIEW

**Comment by jancborchardt** [comment]
Backport at https://github.com/nextcloud/firstrunwizard/pull/47

File: lib/Notification/Notifier.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public function prepare(INotification $notification, $languageCode) {
 				}
 
 				$notification->setParsedSubject($subject)
-					->setLink($this->url->getAbsoluteURL('index.php/settings/personal'));
+					->setLink($this->url->getAbsoluteURL('index.php/settings/user'));
```
Comment (nickvergessen) [inline lib/Notification/Notifier.php]: Should use `getAbsoluteRouteURL()` in first place

Comment (jancborchardt) [inline lib/Notification/Notifier.php]: So just change it from getAbsoluteURL to getAbsoluteRouteURL? Or more? Could you add a commit for that?

Comment (blizzz) [reply lib/Notification/Notifier.php]: @nickvergessen i guess you refer to ``linkToRouteAbsolute()``?  

@jancborchardt then it should be ``$this-url->linkToRouteAbsolute('settings.PersonalSettings.index')``

**Comment by MorrisJobke** [comment]
I also fixed the tests ;)

**Pull request merged**